torch/_subclasses/fake_tensor.py

```diff
@@ -713,10 +713,10 @@ def __new__(
 
         if (
             device.type
-            in ["cuda", "hpu", "xpu", torch._C._get_privateuse1_backend_name()]
+            in ["cuda", "hpu", "xpu", "mps", torch._C._get_privateuse1_backend_name()]
             and device.index is None
         ):
-            if getattr(torch, device.type).is_initialized():
+            if device.type != "mps" and getattr(torch, device.type).is_initialized():
```

Comment:
what's going on here?

Reply:
Is it documented anywhere, that `torch.device.is_initialized()` should be available?  I can implement that function, but I'm unsure what it actually means. That memory were allocated on GPU?

Reply:
I mean, we have torch.cuda.is_initialized which means the CUDA context has been initialized. If your accelerator has no meaningful concept of initialization you can just always return True.